### ISSUE TITLE
ATTACHING FILES ERROR

### ISSUE THREAD

**Issue by Jacquitaloquita** [fluentform/fluentform#312]
Hello I cant attach files to my upload function this is the error message 

<html><header><title>Fatal Error </title></header> <style> .error_content{ background: ghostwhite; vertical-align: middle; margin:0 auto; padding:10px; width:50%; } .error_content label{color: red;font-family: "Ubuntu Mono", Consolas, monospace;font-size: 16pt;font-style: italic;} .error_content ul li{ background: none repeat scroll 0 0 FloralWhite; border: 1px solid AliceBlue; display: block; font-family: "Ubuntu Mono", Consolas, monospace; padding: 2%; text-align: left; } </style> <body style="text-align: center;"> <div class="error_content"> <label >Fatal Error </label> <ul> <li><b>Line</b> 65</li> <li><b>Message</b> Uncaught Error: Class 'FluentValidator\Validator' not found in /var/www/wp-content/plugins/fluentformpro/src/Uploader.php:65 Stack trace: #0 /var/www/wp-content/plugins/fluentformpro/fluentformpro.php(397): FluentFormPro\Uploader->upload() #1 /var/www/wp-includes/class-wp-hook.php(308): FluentFormPro->{closure}('') #2 /var/www/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array) #3 /var/www/wp-includes/plugin.php(517): WP_Hook->do_action(Array) #4 /var/www/wp-admin/admin-ajax.php(188): do_action('wp_ajax_fluentf...') #5 {main} thrown</li> <li><b>File</b> /var/www/wp-content/plugins/fluentformpro/src/Uploader.php</li> </ul> <a href="javascript:history.back()"> Back </a> </div> </body></html>

**Comment by nkb-bd** [comment]
Are you trying to upload files in using file upload/image upload filed? We could not recreate the issue, let us know which version of the plugin you are using free & pro, you should use the latest version of both. Also, purge all of your caches. Let us know what happens.
Thanks

**Comment by techjewel** [comment]
Most probably you are using and OLD version of Fluent Forms Pro. Please update to the latest version.

**Comment by Jacquitaloquita** [comment]
Hello, so thank you for you response. Currently i cannot update the pro
version: i get this message, what can i do? thank you
 [image: image.png]

**Comment by Jacquitaloquita** [comment]
![image](https://github.com/fluentform/fluentform/assets/136031707/b9aa84d6-38c9-4883-84b2-e3beafcc381c)

**Comment by nkb-bd** [comment]
Can you manually download the plugin from your [https://wpmanageninja.com/](https://wpmanageninja.com/) account and upload the plugin?

**Comment by Jacquitaloquita** [comment]
So i get this banner that´s says fluent form pro not working and needs to update and at the same time it updated

![image](https://github.com/fluentform/fluentform/assets/136031707/2d3c07cb-ce86-48cc-a8ad-8d42137135b7)
![image](https://github.com/fluentform/fluentform/assets/136031707/6757897b-b878-4219-9af2-3a58a336fc21)
 I tried deactivating it but it wont work

**Comment by techjewel** [comment]
Fluent Forms Pro needs to be updated to version 5.0 
If you can't update from your site please download the latest version from https://wpmanageninja.com/account/dashboard/ and then install.
